resources.yaml

```diff
@@ -67,7 +67,7 @@ deployment:
       size: 1024
       type: default
   worker-c28m475-htcondor-secondary:
-    count: 8 #19
+    count: 10 #19
```

Comment:
Where are the other 9 ?

Reply:
The numbers are a bit off. We have adjusted and redistributed the flavors. I have asked Manuel the same question and have not received an update yet. I will check with him again.